labelbox/data/serialization/ndjson/label.py

```diff
@@ -227,17 +232,38 @@ def _create_video_annotations(
                 yield NDObject.from_common(annotation=annot, data=label.data)
 
         for annotation_group in video_annotations.values():
-            segment_frame_ranges = cls._get_segment_frame_ranges(
-                annotation_group)
             if isinstance(annotation_group[0], VideoClassificationAnnotation):
-                annotation = annotation_group[0]
```

Comment:
@vbrodsky This line here is were the bulk of the problem comes from on prior solution it is take the first label of the group with its answer and then adding the frames to that. That is why it shows one answer for the entire video

Reply:
Thanks, @Gabefire, for another insightful observation!

I'd like to propose an alternative method to address this issue: by assigning a new segment to each frame that lacks one, in addition to implementing a fix for the issue you are pointing out above. It's important to note that our current approach groups all frames without a specified segment into a single, default segment.

This comment is meant to provide a comprehensive view of the situation. I've initiated a separate story and have already informed @adrian-chang about it. This approach, while potentially beneficial, would require careful consideration and likely some time to resolve the questions raised in my comments on the previous PR.

Reply:
Hey Val! Sounds good! Let me know if you need anything. It was fun looking through this!